empiric-package/empiric/publisher/client.py

```diff
@@ -1,22 +1,31 @@
+from typing import Optional
+
+from empiric.core import LOGGER
 from empiric.core.base_client import EmpiricAccountClient, EmpiricBaseClient
-from empiric.core.const import PUBLISHER_REGISTRY_ADDRESS
+from empiric.core.config import get_config
 from empiric.core.entry import serialize_entries, serialize_entry
+from empiric.core.types import ADDRESS, HEX_STR, TESTNET, Network
 
 
 class EmpiricPublisherClient(EmpiricBaseClient):
+    publisher: Optional[ADDRESS]
+    publisher_registry_address: ADDRESS
+    account_client: EmpiricAccountClient
+
     def __init__(
         self,
         publisher_private_key,
         publisher_address,
-        publisher=None,
-        publisher_registry_address=None,
-        network=None,
-        oracle_controller_address=None,
+        publisher: Optional[ADDRESS] = None,
+        publisher_registry_address: Optional[ADDRESS] = None,
+        network: Network = TESTNET,
```

Comment:
This works for now, in the future we will likely not want to default to TESTNET, but rather force the user to set this